GetSwaggerUI/index.ts

```diff
@@ -10,7 +10,7 @@ const swaggerJsdocUI: AzureFunction = async function (context: Context): Promise
   if (filename == null) {
     context.res = {
       headers: {
-        location: `${trimTrailingSlash(context.req.url)}/index.html`,
+        location: `${trimTrailingSlash(context?.req?.url)}/index.html`,
```

Comment:
context won't ever be nullish, so the first optional chaining operator is redundant